init.lua

```diff
@@ -1,3 +1,15 @@
+-- check for MineClone2
+local mcl = minetest.get_modpath("mcl_core")
+
+if mcl then
+    sounditem = mcl_sounds.node_sound_wood_defaults()
+    item1 = "mcl_core:water_source"
+    stairs = mcl_stairs
```

Comment:
Please don't define global variables. This might cause side-effects in other mods.